demo-transfer-dashboard/src/Components/Form.jsx

```diff
@@ -99,42 +130,44 @@ const Form = () => {
       )}
       {loadingMessage && (
         <p id="loadingMessage" className="loading">
-          <GcdsText>Processing transfer... Please wait.</GcdsText>
+          <GcdsText>{loadingMessage}</GcdsText>
         </p>
       )}
 
-      <table>
-        <thead>
-          <tr>
-            <th>
-              <GcdsText>Patient Name</GcdsText>
-            </th>
-            <th>
-              <GcdsText>Origin PT</GcdsText>
-            </th>
-            <th>
-              <GcdsText>Receiving PT</GcdsText>
-            </th>
-            <th>
-              <GcdsText>Transfer Status</GcdsText>
-            </th>
-          </tr>
-        </thead>
-        <tbody>
-          {transfers.map((transfer, index) => (
-            <tr key={index}>
-              <td>{transfer.patientName}</td>
-              <td>{transfer.originPT}</td>
-              <td>{transfer.receivingPT}</td>
-              <td
-                className={`status ${transfer.status === 'Transferred' ? 'transferred' : 'failed'}`}
-              >
-                <GcdsText>{transfer.status}</GcdsText>
-              </td>
+      {transfers.length > 0 && (
+        <table style={{ marginTop: '50px' }}>
+          <thead>
+            <tr>
+              <th>
+                <GcdsText>Patient Name</GcdsText>
+              </th>
+              <th>
+                <GcdsText>Origin PT</GcdsText>
+              </th>
+              <th>
+                <GcdsText>Receiving PT</GcdsText>
+              </th>
+              <th>
+                <GcdsText>Transfer Status</GcdsText>
+              </th>
             </tr>
-          ))}
-        </tbody>
-      </table>
+          </thead>
+          <tbody>
+            {transfers.map((transfer, index) => (
+              <tr key={index}>
+                <td>{transfer.patient_id}</td>
+                <td>{transfer.originPT}</td>
+                <td>{transfer.receivingPT}</td>
+                <td
+                  className={`status ${transfer.status === 'Transferred' ? 'transferred' : 'failed'}`}
+                >
+                  <GcdsText>{transfer.status}</GcdsText>
```

Comment:
The body of the responses to `GET /transfer-request` on the transfer-outbound API looks like 
```
{
    state,
    finished_on,
    failed_reason,
    patient_id,
    transfer_to,
    stage, 
    completed_stages,
  }
```

See the type information of `get_transfer_request_job_info` in `transfer-outbound/src/transfer_request_queue/transfer_request_utils.ts` for additional info.

Origin PT isn't included currently because that can/should be assumed based on the PT whose API the request was served by. See my comment  in `fetchTransfers`.